aptos-move/move-examples/token_objects/sources/aptos_token.move

```diff
@@ -0,0 +1,841 @@
+        mutable_name: bool,
+        mutable_uri: bool,
+        burnable_by_creator: bool,
+        property_keys: vector<String>,
+        property_types: vector<String>,
+        property_values: vector<vector<u8>>,
+        mutable_properties: bool,
+    ) {
+        let constructor_ref = mint_internal(
+            creator,
+            collection,
+            description,
+            name,
+            uri,
+            mutable_description,
+            mutable_name,
+            mutable_uri,
+            burnable_by_creator,
+            property_keys,
+            property_types,
+            property_values,
+            mutable_properties,
+        );
+
+        let transfer_ref = object::generate_transfer_ref(&constructor_ref);
+        object::disable_ungated_transfer(&transfer_ref);
+    }
+
+    fun mint_internal(
+        creator: &signer,
+        collection: String,
+        description: String,
+        name: String,
+        uri: String,
+        mutable_description: bool,
+        mutable_name: bool,
+        mutable_uri: bool,
+        burnable_by_creator: bool,
+        property_keys: vector<String>,
+        property_types: vector<String>,
+        property_values: vector<vector<u8>>,
+        mutable_properties: bool,
+    ): ConstructorRef {
+        let constructor_ref = token::create(
+            creator,
+            collection,
+            description,
+            name,
+            option::none(),
+            uri,
+        );
+
+        let object_signer = object::generate_signer(&constructor_ref);
+        let mutator_ref = if (mutable_description || mutable_name || mutable_uri) {
+            option::some(token::generate_mutator_ref(&constructor_ref))
+        } else {
+            option::none()
+        };
+
+        let burn_ref = if (burnable_by_creator) {
+            option::some(token::generate_burn_ref(&constructor_ref))
+        } else {
+            option::none()
+        };
+
+        let property_mutator_ref = if (mutable_properties) {
+            option::some(property_map::generate_mutator_ref(&constructor_ref))
+        } else {
+            option::none()
+        };
+
+        let aptos_token = AptosToken {
+            burn_ref,
+            transfer_ref: option::none(),
+            mutator_ref,
+            property_mutator_ref,
+            mutable_description,
+            mutable_name,
+            mutable_uri,
+            freezable_by_creator: false,
+        };
+        move_to(&object_signer, aptos_token);
+
+        let properties = property_map::prepare_input(property_keys, property_types, property_values);
+        property_map::init(&constructor_ref, properties);
+
+        constructor_ref
+    }
+
+    // Accessors
+
+    inline fun borrow<T: key>(token: &Object<T>): &AptosToken {
+        let token_address = object::object_address(token);
+        assert!(
+            exists<AptosToken>(token_address),
+            error::not_found(ETOKEN_DOES_NOT_EXIST),
```

Comment:
nits: 
1. As with other PRs, `not_found` and `already_exists` express the meaning of non-existence or existence. We may want to shorten the Error code name.
2. nites:  reuse the code with `borrow_mut` please.